app/models/repository.rb

```diff
@@ -6,4 +6,8 @@ class Repository < ApplicationRecord
   def to_s
     link
   end
+
+  def name
+    self.decorate.name
+  end
```

Comment:
this should no be in the model, the idea of a decorator is to decorate the original method